src/Persistence.Tests/_TestData/ValidYaml/Group/with-two-children.pa.yaml

```diff
@@ -0,0 +1,7 @@
+Group: 
+Name: My Small Group
+Children:
+- Button:
```

Comment:
Is this correct identation for children items? Screenshoto of an example from new schema.
![image](https://github.com/microsoft/PowerApps-Tooling/assets/10679715/f8f6f4c2-99db-4825-a12f-15822aa72df2)

Reply:
it is correct as of now, but we will change all places at once if/when we change schema